frontend/src/services/authService.js

```
@@ -1,58 +1,238 @@
+      // Update stored user data
+      if (response.data.user) {
+        TokenManager.setTokens(
+          TokenManager.getAccessToken(),
+          TokenManager.getRefreshToken(),
+          response.data.user
+        );
+      }
+
+      return response.data.user;
     } catch (error) {
       throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)